### ISSUE TITLE
Add SVG support

### ISSUE THREAD

**Issue by ZebraFlesh** [Glavin001/atom-preview#128]
This is hardly the "ultimate previewer of source code" if it doesn't have SVG support. Either that or change the tag line so people aren't mislead.

**Comment by Glavin001** [comment]
I agree, SVG support would be great! Please submit a Pull Request for such a feature, I would be happy to review and merge it.

The applicable file to change is https://github.com/Glavin001/atom-preview/blob/master/lib/renderer.coffee#L44 and you could take a look at https://github.com/josa42/atom-svg-preview to see how another package does it.
